projects/components/src/table/cells/data-renderers/enum/string-enum-table-cell-renderer.component.ts

```diff
@@ -0,0 +1,27 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { TableCellRenderer } from '../../table-cell-renderer';
+import { TableCellRendererBase } from '../../table-cell-renderer-base';
+import { CoreTableCellParserType } from '../../types/core-table-cell-parser-type';
+import { CoreTableCellRendererType } from '../../types/core-table-cell-renderer-type';
+import { TableCellAlignmentType } from '../../types/table-cell-alignment-type';
+
+@Component({
+  selector: 'ht-string-enum-table-cell-renderer',
+  styleUrls: ['./string-enum-table-cell-renderer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div
+      [ngClass]="{ clickable: this.clickable, 'first-column': this.isFirstColumn }"
+      class="string-enum-cell"
+      [htTooltip]="this.value"
+    >
+      {{ this.value | htDisplayStringEnum }}
+    </div>
+  `
+})
+@TableCellRenderer({
+  type: CoreTableCellRendererType.StringEnum,
+  alignment: TableCellAlignmentType.Left,
+  parser: CoreTableCellParserType.String
+})
+export class StringEnumTableCellRendererComponent extends TableCellRendererBase<string> {}
```

Comment:
nit: IMO, the name 'String Enum' is not very clear. Or we can at least add similar comments from `htDisplayStringEnum` here too

```
  // This removes dashes and underscores and gives all words initial caps

  // We only want the first word capitalized.
```

Reply:
I always struggle with names, but also always open to suggestions.

I've added the comment in my local. That should help some. Not worth holding up this PR for, but I'll commit them in whatever next HT PR I have.